usr/share/rear/finalize/GNU/Linux/320_migrate_network_configuration_files.sh

```diff
@@ -3,12 +3,11 @@
 
 # because the bash option nullglob is set in rear (see usr/sbin/rear)
 # PATCH_FILES is empty if nothing matches $TARGET_FS_ROOT/etc/sysconfig/*/ifcfg-*
-PATCH_FILES=( $TARGET_FS_ROOT/etc/sysconfig/*/ifcfg-* )
+PATCH_FILES=( $TARGET_FS_ROOT/etc/sysconfig/*/ifcfg-* $TARGET_FS_ROOT/etc/network/inter[f]aces $TARGET_FS_ROOT/etc/network/interfaces.d/* )
```

Comment:
Why do you use globbing for a single character as in inter[f]aces?
I think globbing for a single character is useless because it matches
only this single character (it is neither case-insensitive nor optional):
<pre>
# touch /tmp/bar
# touch /tmp/bAr
# touch /tmp/br

# echo /tmp/b[a]r
/tmp/bar

# ( shopt -s nullglob extglob ; echo /tmp/b[a]r )
/tmp/bar
</pre>

Reply:
I thought it also permit to take only the file if it exists on the server. 
But maybe I misunderstood the comment:
```
# because the bash option nullglob is set in rear (see usr/sbin/rear)
# PATCH_FILES is empty if nothing matches $TARGET_FS_ROOT/etc/sysconfig/*/ifcfg-*
```
I just don't want to add /etc/network/interfaces to the PATCH_FILES array if it does not exists (like suse/redhat)

Reply:
Perfect!
Thanks for the explanation.
You are right:
<pre>
# arr=( foo /etc/network/inter[f]aces bar ) ; echo ${arr[*]}
foo /etc/network/inter[f]aces bar

# ls /etc/network/interfaces
ls: cannot access /etc/network/interfaces: No such file or directory

# ( shopt -s nullglob extglob ; arr=( foo /etc/network/inter[f]aces bar ) ; echo ${arr[*]} )
foo bar
</pre>

Reply:
Perhaps an additional comment why
"artificial" looking globbing is used?

Reply:
ok